t/t0052-simple-ipc.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
+
+test_description='simple command server'
```

Comment:
It probably makes more sense to squash this into the same commit that introduces the test helper: one is useless without the other.

Reply:
sure